MonstieWash/Assets/Scripts/Video/VideoController.cs

```diff
@@ -105,7 +108,13 @@ private void PlayTitleVideo()
     /// Sets correct settings and plays animatic.
     /// </summary>
     public void PlayAnimatic()
-    {      
+    {
+        // disable all forms of player input during animatic
+        InputSystem.DisableDevice(Keyboard.current);
+        InputSystem.DisableDevice(Mouse.current);
+        m_inputManager.Disable();
```

Comment:
The Input System already has a way to do all this unfortunately. You can pretty much remove all the code you've added (sorry!) and just replace this part with the following and it should do what you're after in a much tidier way:

```suggestion
        // disable all forms of player input during animatic
        InputManager.Instance.SetControlScheme(InputManager.ControlScheme.None);
```